src/pages/GetStarted/GetStarted.tsx

```diff
@@ -1,10 +1,22 @@
 import React from 'react';
 import { NavButton } from 'components/NavButton/NavButton';
+import { InstructionModal } from 'components/InstructionModal/InstructionModal';
+import backgroundArrow from '../../assets/background-arrow.svg';
+import './GetStarted.css';
+
+const button: Button = {text: "GET STARTED", to: "/AddPhoto" };
+const intro: InstructModal = {text: ["Let’s Make a \npattern in 3 \nsteps"]};
+const steps: InstructModal = {text: ["1. Upload photo", "2. Trace pattern", "3. Print pdf"]};
 
-const button: Button = {text: "Get Started", to: "/AddPhoto" };
 
 export const GetStarted: React.FC = () => {
-    return (
-        <NavButton button={button}/>
-    );
+    return (<>
+        <div className={'backgroundImageTop'}></div>
+        <img className={'backgroundImageBottom'} src={backgroundArrow}></img>
```

Comment:
maybe adding an alt prop to the img component?

Reply:
I saw that recommendation from the linter. The image is just the grey arrow in the background so its not very important, but I will add some alt text anyway.